### DISCUSSION TITLE
Error message when running Prody in VMD 1.9.4a53 Windows 10

### DISCUSSION THREAD

**Discussion by tanoury1** [prody/ProDy#2169] [Archived Issues]
Hi,
I tried to run Prody from VMD.  When I submitted the job, I got the initial message saying it could not find prody.  I selected "Yes" to point to the path.  My path is:
C:\Users\tanoury\Miniconda3\pkgs\prody-2.2.0-py38h885f38d_0\Scripts\prody.exe

When the job tries to run again, I get the following error message:
    while executing
"animate write pdb $pdbfn beg $::NMWiz::prodyFirstFrame end $end skip $::NMWiz::prodySkipFrame waitfor all sel $sel $::NMWiz::prodyMolid"
    (procedure "::NMWiz::prodySubmitPCAjob" line 43)
    invoked from within
"::NMWiz::prodySubmitPCAjob"
    (procedure "::NMWiz::prodySubmitJob" line 26)
    invoked from within
"::NMWiz::prodySubmitJob"
    invoked from within
".nmwizprody.submitFrame.prodySubmit invoke"
    ("uplevel" body line 1)
    invoked from within
"uplevel #0 [list $w invoke]"
    (procedure "tk::ButtonUp" line 24)
    invoked from within
"tk::ButtonUp .nmwizprody.submitFrame.prodySubmit"
    (command bound to event)

Thank you for your help,
Jerry

**Comment by jamesmkrieger** [comment]
This seems like a vmd problem not a prody problem. Perhaps you could ask the vmd people as described at https://www.ks.uiuc.edu/Research/vmd/mailing_list/. I suspect it's something to do with not being able to write files.

If you describe what you're doing and what is happening in more detail, we can maybe help to figure it out too.

**Comment by jamesmkrieger** [comment]
Could you perhaps try an earlier version of vmd? It could be that nmwiz has some command that isn't compatible with the latest one
